### PR TITLE
Fixes cult datum giving the action button to the mind instead of mob

### PR DESCRIPTION
**Pull request by lzimann**
Also fixes trying to replace the mind instead of the body when the player is jobbaned.

### CODE REVIEW

File: code/datums/antagonists/datum_cult.dm [hunk 1]
```diff
@@ -10,20 +10,20 @@
 	if(!owner)
 		return
 	if(jobban_isbanned(owner.current, ROLE_CULTIST))
-		addtimer(CALLBACK(SSticker.mode, /datum/game_mode.proc/replace_jobbaned_player, owner, ROLE_CULTIST, ROLE_CULTIST), 0)
+		addtimer(CALLBACK(SSticker.mode, /datum/game_mode.proc/replace_jobbaned_player, owner.current, ROLE_CULTIST, ROLE_CULTIST), 0)
 	owner.current.log_message("<font color=#960000>Has been converted to the cult of Nar'Sie!</font>", INDIVIDUAL_ATTACK_LOG)
 
 /datum/antagonist/cult/apply_innate_effects()
```
Comment (ChangelingRain) [inline code/datums/antagonists/datum_cult.dm]: where's the mob/living/mob_override bit? you neeeeeeed that.

Comment (lzimann) [inline code/datums/antagonists/datum_cult.dm]: bleh

File: code/datums/antagonists/datum_cult.dm [hunk 2]
```diff
@@ -10,24 +10,24 @@
...
+		addtimer(CALLBACK(SSticker.mode, /datum/game_mode.proc/replace_jobbaned_player, owner.current, ROLE_CULTIST, ROLE_CULTIST), 0)
 	owner.current.log_message("<font color=#960000>Has been converted to the cult of Nar'Sie!</font>", INDIVIDUAL_ATTACK_LOG)
 
-/datum/antagonist/cult/apply_innate_effects()
+/datum/antagonist/cult/apply_innate_effects(mob/living/mob_override)
 	. = ..()
 	owner.current.faction |= "cult"
 	owner.current.verbs += /mob/living/proc/cult_help
-	communion.Grant(owner)
+	communion.Grant(owner.current)
```
Comment (ChangelingRain) [inline code/datums/antagonists/datum_cult.dm]: and you _do_ need to actually use mob_override for this, too.

Comment (lzimann) [inline code/datums/antagonists/datum_cult.dm]: bleh

**Comment by Robustin** [comment]
Cult (and my PR) are broken until this is merged in case that wasn't clear.

**Pull request merged**